webpack.common.js

```diff
@@ -0,0 +1,39 @@
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { resolve } = require('path');
+
```

Comment:
@manedurphy this is cool! So is the css and html plugin the only difference between the standard webpack we've been using and the prod? Do you think the pages would render the same way if someone else was just using regular webpack or should we all use the prod version?

Reply:
I think we are all going to end up having a production configuration where the first step is to set the mode property to "production". This way we will see warnings when our bundle sizes exceed a certain threshold. I currently have it set such that the HtmlWebpackPlugin targets a template html file in my Proxy repo (in the public folder). That's just the template that webpack references when I run the build script. I have two root divs in their with unique IDs, where my first service will be renders in "root", and my second service will be rendered in "root2". When I clone everyone else's repo, I will add more root divs  and adjust the target ID that react-dom inserts each service in to.